### ISSUE TITLE
New 0.1.0 release Feedback

### ISSUE THREAD

**Issue by MayeulC** [MayeulC/Saitek#9]
Hi everyone, I put a new release online.
As explained in the [release announcement](https://github.com/MayeulC/Saitek/releases/tag/v0.1.0) (I think it is worth reading, or at least skimming), I would like to gather some feedback, both positive and negative, on the new release.

Feel free to open some issues if you have some device-specific requests, for example.

People who could be interested in the new release: 
@kc14, @sorinn, @Milkman00, @MaikuMori, @angelus2014, @alphaphee, @noobish, @np, @improti

I hope I didn't forget anyone, this is almost everyone who contributed to the project and/or had issues with their mouse.

Have a nice day !

**Comment by alphaphee** [comment]
Hey @MayeulC, great work! Just noticed the release, everything works great with my RAT9. If you want any help with the gui design of future releases please let me know, I would be happy to help.

_Edit_: The configurator did not save my changes, the next time I launched it presented the original values once again.

**Comment by MayeulC** [comment]
Hi @alphaphee, thank you for your feedback, I just confirmed that the script wasn't working anymore... Not even changing tabs, which should change the mouse mode.

I also tried some previous versions, and it seems to be broken as well, so the breakage was likely introduced somewhere else in the meantime (especially since I wouldn't have pushed some untested code). The likely culprit is libusb. I will have to debug this, and maybe change the backend for another library.

As for gui design and future releases, definitely! But I am working on and off on it at the moment, so don't expect any scedule there 😆
